app/src/nxt/res/layout/fragment_nxt_firmware.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't know but the appearance is bit narrow, isn't it?
(and the direction of the button is strange for me; vertical seems natural.)